cmd/cloud/cluster.go

```diff
@@ -247,6 +265,33 @@ var clusterListCmd = &cobra.Command{
 	},
 }
 
+var clusterUtilitiesCmd = &cobra.Command{
+	Use:   "utilities",
+	Short: "Show metadata regarding utility services running in a cluster.",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+		clusterID, err := command.Flags().GetString("cluster")
+		if err != nil {
+			return err
+		}
+
+		metadata, err := client.GetClusterUtilities(clusterID)
+		if err != nil {
+			return err
+		}
+
+		err = printJSON(metadata)
+		if err != nil {
+			return err
+		}
```

Comment:
Yeah, easier said than done (in a robust way) because of the way Helm reports versions :\

Reply:
Yeah, agreed. Helm 3 and the Go library may be our path forward to that.

Reply:
Yeah and at the very least this change should make the transition to that easier.

Reply:
Actually thinking about it a little more, this shouldn't be too bad because of the way I've laid out the interface.. I should be able to use string manipulation to extract the version from any given chart because it can be a utility-specific implementation.. so that shouldn't be brittle. I will go ahead and do this so that the result in the JSON struct is formatted the same and if/when we want we can just compare those fields on the client side.

tl;dr after thinking about it, I can reliably format actualVersions and desiredVersions identically without too much trouble or creating brittle code, so I will do that.

Reply:
Updated the implementation for each Utility to have the method ActualVersion() strip the Helm chart name prefix from each Utility when returning the value.

I realized after you brought this up that this should be perfectly reliable since it can vary for each Utility and it should be easy to continue the pattern if/when we move to Helm 3 and/or Operators, even if we make that change piecemeal:

https://github.com/mattermost/mattermost-cloud/pull/143/commits/606ab4da45091dc2403ded69473b1d9bbe9769d1

I was originally worried that doing this would be brittle but since it can be implemented specifically for each utility, this should be perfectly robust.